web-registry/src/pages/Marketplace/Storefront/Storefront.tsx

```diff
@@ -58,28 +58,49 @@ import {
   normalizeSellOrders,
 } from './Storefront.normalizer';
 import { SellOrderActions } from './Storefront.types';
-import {
-  getCancelCardItems,
-  sortBySellOrderId,
-  updateBatchInfosMap,
-  updateProjectsWithMetadataMap,
-} from './Storefront.utils';
+import { getCancelCardItems, sortBySellOrderId } from './Storefront.utils';
 
 export const Storefront = (): JSX.Element => {
+  const client = useApolloClient() as ApolloClient<NormalizedCacheObject>;
+  const { ecocreditClient } = useLedger();
+
   const [paginationParams, setPaginationParams] =
     useState<TablePaginationParams>({
       page: 0,
       rowsPerPage: DEFAULT_ROWS_PER_PAGE,
       offset: 0,
     });
   const { offset, rowsPerPage } = paginationParams;
-  const { sellOrdersResponse, refetchSellOrders } = useQuerySellOrders();
-  const sellOrders = sellOrdersResponse?.sellOrders;
+
+  // We do not use pagination yet on the SellOrders query
+  // because the ledger currently returns sell orders ordered by seller address and not by id
+  // which would result in a list of non sequential ids in the sell orders table and look weird
+  // TODO: use pagination as soon as this is fixed on Regen Ledger side (as part of v5.0)
+  const { sellOrders, refetchSellOrders } = useFetchSellOrders();
   const uiSellOrdersInfo = useMemo(
     () => sellOrders?.map(normalizeToUISellOrderInfo),
     [sellOrders],
   );
 
+  // Off-chain stored Projects
```

Comment:
Out of the scope of this but it might be worth grouping all these requests into a hook (like `useProjectsWithMetadata`) later on.